src/components/layout/hero/Main.jsx

```diff
@@ -44,6 +44,7 @@ const Main = () => {
   useEffect(() => {
     getHero();
     getEvents();
+    getDictionary();
```

Comment:
dlaczego getDictionary w tym miejscu? Dla init, zawsze musi być to zawsze główny komponent